tensilelite/Tensile/Source/client/include/DataInitialization.hpp

```diff
@@ -591,6 +611,18 @@ namespace Tensile
                 case InitMode::TrigAbsCos:
                     initArrayTrig<T, true, true>(array, tensor);
                     break;
+                case InitMode::TrigIndSin:
```

Comment:
TrigInd is problem independent trig initialization, so that we don't need to re-initialize every time we change a problem during tuning.
TrigIndSign goes to initArray(InitMode mode, T* array, size_t elements), but we still need these codes, otherwise, compiler will report error.